protocols/kad/src/behaviour.rs

```diff
@@ -116,6 +117,9 @@ pub struct Behaviour<TStore> {
 
     /// The record storage.
     store: TStore,
+
+    /// The interval used by [`Behaviour::poll`] to call [`Behaviour::bootstrap`].
+    refresh_interval: Option<Duration>,
```

Comment:
Lets stick to one terminology:

```suggestion
    bootstrap_interval: Option<Duration>,
```

Reply:
We will need to store a `Delay` as well that we can poll and it needs to be reset every time it resolves.

Reply:
I am storing the delay in the `Behaviour` object